code/modules/atmospherics/machinery/airalarm.dm

```diff
@@ -722,7 +728,7 @@
 		return
 
 	var/datum/signal/alert_signal = new(list(
-		"zone" = get_area_name(src),
+		"zone" = A,
```

Comment:
Are you sure?

Reply:
Yes

Reply:
it will result in the name anyway

Reply:
and also without the actutal area i cant get the atmos console to work

Reply:
John helped me gets this to work with the area instead of the name, this will be more accurate